test/acceptance/application-test.js

```diff
@@ -34,6 +34,11 @@ describe('Acceptance | application', function() {
         fixturesPath: 'tests'
       });
     }).then(function() {
+      app.editPackageJSON(function(pkg) {
+        pkg.devDependencies['ember-cli-fastboot'] = process.env.npm_package_devDependencies_ember_cli_fastboot;
```

Comment:
Ok, regarding the env var, was not aware of the magic npm makes there, just learned about this. 👨‍🎓

Regarding the fastboot install, could you maybe add a comment stating the reasoning for that, so anybody else who looks at this does not misinterpret that as useless (just as I did)?